src/Sulu/Bundle/MediaBundle/Resources/js/stores/MediaUploadStore/MediaUploadStore.js

```diff
@@ -1,36 +1,18 @@
 // @flow
-import {action, computed, observable} from 'mobx';
-import {ResourceStore, ResourceMetadataStore} from 'sulu-admin-bundle/stores';
+import {action, observable} from 'mobx';
+import {ResourceMetadataStore} from 'sulu-admin-bundle/stores';
 import {ResourceRequester} from 'sulu-admin-bundle/services';
 
 const RESOURCE_KEY = 'media';
-const THUMBNAIL_SIZE = 'sulu-400x400-inset';
 
 export default class MediaUploadStore {
-    resourceStore: ResourceStore;
+    locale: observable;
```

Comment:
That will probably fail now in flow... check #3674 to see how to fix that.

Reply:
It didn't but still changed it to `IObservableValue`

Reply:
It probably does with the newer version of `mobx`. You've probably installed an old one.